test/functional/admin/generic_editions_controller_tests/political_document_test.rb

```diff
@@ -2,18 +2,32 @@
 
 class Admin::GenericEditionsController::PolticalDocumentsTest < ActionController::TestCase
   tests Admin::NewsArticlesController
+  enable_url_helpers
 
   setup do
     login_as :writer
   end
 
-  view_test "displays the political checkbox for privileged users " do
-    create(:current_government)
+  view_test "displays the history mode form control for managing editor users" do
+    current_government = create(:current_government)
     login_as :managing_editor
     published_edition = create(:published_news_article, first_published_at: 2.days.ago)
     new_draft = create(:news_article, document: published_edition.document, first_published_at: 2.days.ago)
     get :edit, params: { id: new_draft }
-    assert_select "#edition_political"
+    assert_select "#edition_political input[type='checkbox'][name='edition[government_id]'][value='#{current_government.id}']"
+  end
+
+  # TODO: Decide whether to move this into an edition test helper used by all edition types
```

Comment:
It looks fine where it is to me - what benefit are you thinking moving it to an edition test helper would give us?